ONSdigital/dp-design-system
# removed the hardcoded search path.

### What

 This allows the code to be reused for /census/find-a-dataset

### How to review

- Check that it doesn't break any of the search functionality
- Check new endpoint works properly. https://github.com/ONSdigital/dp-frontend-search-controller/pull/199

### Who can review

!me
